tests/unit/test-util/test.adb.js

```diff
@@ -242,11 +258,25 @@ describe('utils/adb', () => {
           }),
         },
       });
+
+      const stubGetCurrentUser = sinon.stub(
+        ADBUtils.prototype,
+        'getCurrentUser',
+      );
+      stubGetCurrentUser.resolves(0);
+
       const adbUtils = new ADBUtils({ adb });
 
       const promise = adbUtils.discoverInstalledFirefoxAPKs('device1');
       const packages = await assert.isFulfilled(promise);
       sinon.assert.calledOnce(adb.fakeADBDevice.shell);
```

Comment:
Could you also verify that `stubGetCurrentUser` was calledOnce?

Reply:
Done